python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -355,8 +355,29 @@ def quantize(self, model,  # remove the type requirement for type checking
                                                quantized_onnx_model)
                 else:
                     return return_pl_model
+        elif backend == 'pot' or accelerator == 'openvino':
```

Comment:
shall we just automatically infer the backend: using pot for openvino, and inc for others?